dask/array/tests/test_overlap.py

```diff
@@ -123,6 +123,7 @@ def test_overlap_internal_asymmetric_small():
 def test_trim_internal():
     d = da.ones((40, 60), chunks=(10, 10))
     e = trim_internal(d, axes={0: 1, 1: 2}, boundary="reflect")
+    assert any("_overlap_trim_info" in k[0] for k in dict(e.dask))
```

Comment:
Works for me. FWIW, I was thinking more about a test that confirms the original intent of ensuring that graphs remain small.